frappe/public/js/frappe/form/footer/timeline.js

```diff
@@ -376,7 +376,7 @@ frappe.ui.form.Timeline = class Timeline {
 		c["delete"] = "";
 		c["edit"] = "";
 		if(c.communication_type=="Comment" && (c.comment_type || "Comment") === "Comment") {
-			if(frappe.model.can_delete("Comment")) {
+			if(frappe.user.name == c.sender && frappe.model.can_delete("Comment")) {
```

Comment:
@ronelvcabrera Your second condition is useless because can_delete gives permission to delete to everyone as such who have desk access. This PR will not be accepted in core also. I did something similar and was asked to fix can_delete.
For reference see this: https://github.com/frappe/frappe/pull/8174
Rushabh's comment to fix the root cause: https://github.com/frappe/frappe/pull/8174#issuecomment-526446660

Reply:
> Your second condition is useless because can_delete gives permission to delete to everyone as such who have desk access

First, no, it is only for "Comment" doctype. As you can see.

I did not see the previous PR. I believe what was meant is if this is to push to core, it should be configured on permissions.

But I believe this works fine for our requirement. Please advise to close it or not.

Reply:
@ronelvcabrera Try removing it, it still works fine.